renderlib/gesture/gesture.h

```diff
@@ -397,7 +397,7 @@ struct Gesture
 
     // Gesture draw, called once per window update (frame) when the GUI draw commands
     // had been described in full.
-    void draw(struct SceneView& sceneView, const struct SelectionBuffer* selection);
+    void draw(struct SceneView& sceneView, struct SelectionBuffer* selection);
```

Comment:
what does this change do?

Reply:
The `clear()` method on SelectionBuffer actually modifies some internal data so it can't be considered const.  If a function calls any non-const methods on an object then the object can't be passed in as const.